src/YetAnotherHttpHandler.Unity/Assets/Editor/Scripts/PackageExporter.cs

```diff
@@ -1,34 +1,71 @@
 #if UNITY_EDITOR
 
 using System;
+using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using UnityEditor;
-using UnityEngine;
+using UnityEditor.PackageManager.Requests;
 
 public static class PackageExporter
 {
+    private static ListRequest s_listRequest;
+
     [MenuItem("Tools/Export Unitypackage")]
     public static void Export()
     {
-        PackDependencies("Cysharp.Net.Http.YetAnotherHttpHandler.Dependencies", "YetAnotherHttpHandler.deps.txt");
-        PackDependencies("Grpc.Net.Client.Dependencies", "Grpc.Net.Client.deps.txt");
+        EditorUtility.DisplayProgressBar("Packges", "Exporting packages, please wait...", 0);
+
+        s_listRequest = UnityEditor.PackageManager.Client.List(offlineMode: false, includeIndirectDependencies: true);
+
+        EditorApplication.update += OnEditorUpdate;
+    }
+
+    private static void OnEditorUpdate()
+    {
+        if (s_listRequest == null || !s_listRequest.IsCompleted)
+        {
+            return;
+        }
+
+        EditorApplication.update -= OnEditorUpdate;
+
+        ExportPackage("com.cysharp.yetanotherhttphandler", "Cysharp.Net.Http.YetAnotherHttpHandler.Dependencies", includeSelf: false);
```

Comment:
This also needs to be changed because the referenced package name has changed.
```suggestion
        ExportPackage("com.cysharp.yetanotherhttphandler.dependencies", "Cysharp.Net.Http.YetAnotherHttpHandler.Dependencies", includeSelf: false);
```

Reply:
Otherwise, it looks good. 👍

Reply:
@mayuki done!